### PR TITLE
Created static UI of profile management (#202regg)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

**Comment by azkyakhan** [comment]
profile

File: src/views/Profile.vue [hunk 1]
```diff
@@ -0,0 +1,108 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
```
Comment (dt2patel) [inline src/views/Profile.vue:4]: missing back button

File: src/views/Profile.vue [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        <ion-title>{{ $t("Profile") }}</ion-title>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content :fullscreen="true">
```
Comment (dt2patel) [inline src/views/Profile.vue]: @azkyakhan please make sure to check all PRs for full screen true

File: src/views/Profile.vue [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+      <div>
```
Comment (dt2patel) [inline src/views/Profile.vue]: semantic tag

**Review by dt2patel — CHANGES_REQUESTED** [review]
some changes to UI, please add spacers to reflect similar spacing to Figma

Also I simplified the UI in Figma to help reduct the complexity of code.

File: src/views/Profile.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/Profile.vue:4]: I have added back button.

File: src/views/Profile.vue (hunk 2 above)
Comment (azkyakhan) [reply src/views/Profile.vue]: Removed it sir.

File: src/views/Profile.vue (hunk 3 above)
Comment (azkyakhan) [reply src/views/Profile.vue]: Replaced div tag with `main` tag.

**Comment by azkyakhan** [comment]
profile

**Review by dt2patel — APPROVED** [review]

**Pull request merged**